app/src/server/api/routers/payments.router.ts

```diff
@@ -93,6 +94,20 @@ export const paymentsRouter = createTRPCRouter({
           defaultPaymentMethodId = availablePaymentMethodId;
         }
 
+        // Update Rate Limit after a payment method is added
+        if (paymentMethods.data.length) {
+          const project = await prisma.project.findUniqueOrThrow({
+            where: { id: input.projectId },
+          });
+
+          if (project.rateLimit === 3) {
```

Comment:
Let's define constants for the "3" and "20" and reference them everywhere (including in the documentation) so we can remind ourselves to update them everywhere when we change these limits in the future.